spring-webmvc/src/main/java/org/springframework/web/servlet/config/annotation/PathMatchConfigurer.java

```diff
@@ -126,7 +126,7 @@ public PathMatchConfigurer addPathPrefix(String prefix, Predicate<Class<?>> pred
 	 * @deprecated as of 5.2.4. See class-level note in
 	 * {@link RequestMappingHandlerMapping} on the deprecation of path extension
 	 * config options. As there is no replacement for this method, in 5.2.x it is
-	 * necessary to set it to {@code false}. In 5.3 the default changes to
+	 * necessary to set it to {@code false}. by default it is set to
```

Comment:
The curent javadoc reads ok to me. What lead you to such a change?

Reply:
https://github.com/spring-projects/spring-framework/issues/27269 see this issues

Reply:
I have misunderstand sorry